std/special/compiler_rt.zig

```diff
@@ -505,7 +527,16 @@ nakedcc fn __aeabi_memmove() noreturn {
 
 nakedcc fn __aeabi_memset() noreturn {
     @setRuntimeSafety(false);
-    if (use_thumb_1) {
+    if (use_thumb_1_pre_armv6) {
+        asm volatile (
+            \\ eors    r1, r2
```

Comment:
I've only worked on recent versions of arm and aarch64 so I'm not fully clued up on the intricacies of thumb, but can you use `eor` instead? The only difference AFAIK is that it doesn't update the condition flags. Same goes for `adds` -> `add` and `movs` -> `mov` in `memclr`.

Reply:
I don't think the non-flag-updating versions of these instructions are available in thumbv4t -- [the GBATEK document I'm using as a reference](https://problemkaputt.de/gbatek.htm#thumbinstructionsummary) just calls this instruction `EOR` and says it always updates the condition flags.

Reply:
Yes you're right, I just got an ARMv6t ISA reference card from someone at work and there is no `EOR`, only `EORS`.